packages/react/src/patterns/sub-patterns/Layout/Layout.js

```diff
@@ -116,7 +116,7 @@ const Layout = ({
   <section
     data-autoid={`${stablePrefix}--layout`}
     className={classnames(
-      nested ? '' : `${prefix}--grid`,
+      nested ? `${prefix}--col` : `${prefix}--grid`,
```

Comment:
Why did you make this change? This looked correct before you added `${prefix--col}`. This whole pattern has to sit inside of a `--col`, not be a `--col`.